python-lib/xlsx_writer.py

```diff
@@ -123,30 +168,32 @@ def rename_too_long_dataset_names(input_dataset_names: List[str]) -> Dict[str, s
                 index_rename += 1
                 rename = f"{name[0:renaming_length]}{index_rename:02d}"
 
-            logger.info(f"Dataset {name} with a too long name will be stored as sheet {rename}")
+            logger.info(f"Dataset '{name}' with a too long name will be stored as sheet '{rename}'")
             return_map[name] = rename
         else:
             return_map[name] = name
 
     return return_map
 
-
 def datasets_to_xlsx(input_dataset_names, xlsx_abs_path, worksheet_provider):
     """
-    Write the input datasets into same excel into the folder
-    :param input_dataset_names: the list of dataset to put in a single excel file, using one sheet (excel tab) per dataset
+    Write each input dataset into one temporary excel and merge all these excel into the final excel file
```

Comment:
This method is getting big... Its readability could be improve by splitting into (probably) 4 methods (datasets to workbooks, create template wb file, extract tmp wb into template, zip the final excel file ?).

Reply:
done in 1887f6299b2cb70e601bb9b7a9d89414709aea94